packages/core-components/src/components/TopBar.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { ReactNode } from 'react';
+
+export type TopBarProps = {
+  mobileMode?: boolean;
+  className?: string;
+  brandIcon?: ReactNode;
+  phoneIcon?: ReactNode;
+  languageButton?: ReactNode;
+  linkContact1?: string;
+  linkContact2?: string;
+  phoneNumber1?: string;
+  phoneNumber2?: string;
+  nameContact1?: string;
+  nameContact2?: string;
+};
+
+const TopBar: React.FC<TopBarProps> = ({
+  className,
+  mobileMode = window.innerWidth < 768,
+  brandIcon,
+  phoneIcon,
+  languageButton,
+  linkContact1,
+  linkContact2,
+  phoneNumber1,
+  phoneNumber2,
+  nameContact1,
+  nameContact2,
+}) => {
+  return (
+    <div
+      className={`${className} container sm:bg-white md:bg-primary flex flex-row items-center w-screen`}
+    >
+      {mobileMode && (
```

Comment:
@Neeze delete mobile code, we usually use tailwind to implement responsive layout, if it is not possible, we will need to create another component